app/lib/bk/compat/parsers/bitrise/steps.rb

```diff
@@ -28,6 +59,18 @@ def translate_git_clone(_inputs)
           '# No need for cloning, the agent takes care of that'
         end
 
+        def translate_git_tag(inputs)
+          return '# Invalid git-tag step configuration!' unless inputs.include?('tag')
+
+          cmd_all_params = "git tag -fa #{inputs['tag']} -m #{inputs['tag_message']}"
+          cmd_no_message = "git tag -fa #{inputs['tag']}"
+
+          [
+            inputs['tag_message'].nil? ? cmd_no_message : cmd_all_params,
+            !inputs['push'].nil? && inputs['push'] ? 'git push --tags' : nil
```

Comment:
a simple way to write this would be:

```suggestion
            inputs.fetch('push', false) ? 'git push --tags' : nil
```

Reply:
`Hash#fetch` is much nicer - I actually all similar conditionals to this approach!

Reply:
now that I think about it, if the default value is also false-y, then it should not be needed as `nil` would also map to the second branch of the conditional. As a general rule, `hash.fetch(key, default)` is only a good idea when `default` has a different behaviour than what would cause the key not being present